### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by Sunaries**
- Contributions by @WatchGintama.
- Contributions by @TentuZero 
- Standardized "Petrifying Frost" as a curse.
- Fixed Dust Devil's damage type from Cut to Slash.
- Minor fixes.

### CODE REVIEW

File: EnglishTranslation-master/ITEM.tsv [hunk 1]
```diff
@@ -3816,10 +3816,10 @@ ITEM_20150317_003815	Body of Bishop
 ITEM_20150317_003816	Tasty Candy Ball 
 ITEM_20150317_003817	A tasty candy ball made for the workers at the petrified city in Kaliss. 
 ITEM_20150317_003818	Guard's Memento 
-ITEM_20150317_003819	Memento of a soldier who died to the petrifying frost.  
+ITEM_20150317_003819	Memento of a guard who died to the petrifying frost.
```
Comment (ttgmichael) [inline EnglishTranslation-master/ITEM.tsv]: stone frost? Have we settled with a name for the frost xD?

File: EnglishTranslation-master/QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -2414,20 +2414,20 @@ QUEST_LV_0200_20150317_002413	You've defeated Elite Monsters as requested by Sal
...
+QUEST_LV_0200_20150317_002419	Graverobber Amanda, asked you to hand over the records of Ruklys' era to Wilhelmina Carriot who is working at Kovos Hall. After that, you should receive a weapon from Wilhelmina Carriot which you can use to defeat Gargoyle.
 QUEST_LV_0200_20150317_002420	You've received the vacuum scroll from Wilhelmina Carriot. Let's give it to Graverobber Amanda.
-QUEST_LV_0200_20150317_002421	Defeat Gargoyles
-QUEST_LV_0200_20150317_002422	The thief, Amanda, told you to destroy Gargoyles at the rendezvous point that are trying to enter the Fortress of the Land. Use the vacuum scroll that you received from Wilhelmina Carriot and defeat Gargoyles without making any sound.
-QUEST_LV_0200_20150317_002423	You've defeated Gargoyles as Graverobber Amanda requested to you. Return to her and let's talk about what to do next.
+QUEST_LV_0200_20150317_002421	Defeat Gargoyle
+QUEST_LV_0200_20150317_002422	Graverobber Amanda, told you to destroy Gargoyle at the rendezvous point that are trying to enter the Fortress of the Land. Use the vacuum scroll that you received from Wilhelmina Carriot and defeat Gargoyle without making any sound.
```
Comment (ttgmichael) [inline EnglishTranslation-master/QUEST_LV_0200.tsv:2422]: Lose the comma after Amanda

File: EnglishTranslation-master/QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -2414,20 +2414,20 @@ QUEST_LV_0200_20150317_002413	You've defeated Elite Monsters as requested by Sal
...
 QUEST_LV_0200_20150317_002415	Graverobber Amanda is waiting for someone's help in the Petrified City.
 QUEST_LV_0200_20150317_002416	Collect Documentaries of Ruklys' Era
-QUEST_LV_0200_20150317_002417	The thief, Amanda, asked you to collect the records of the era of Ruklys at the rendezvous point for the transaction with Wilhelmina Carriot.
+QUEST_LV_0200_20150317_002417	Graverobber Amanda, asked you to collect the records of the era of Ruklys at the rendezvous point for the transaction with Wilhelmina Carriot.
 QUEST_LV_0200_20150317_002418	You've collected the records of the era of Ruklys as requested by Amanda. Let's go back to Amanda.
-QUEST_LV_0200_20150317_002419	The thief, Amanda, asked you to hand over the records of Ruklys' era to Wilhelmina Carriot who is working at Kovos Hall. After that, you should receive a weapon from Wilhelmina Carriot which you can use to defeat Gargoyles.
+QUEST_LV_0200_20150317_002419	Graverobber Amanda, asked you to hand over the records of Ruklys' era to Wilhelmina Carriot who is working at Kovos Hall. After that, you should receive a weapon from Wilhelmina Carriot which you can use to defeat Gargoyle.
```
Comment (ttgmichael) [inline EnglishTranslation-master/QUEST_LV_0200.tsv:2419]: Lose the comma after Amanda

File: EnglishTranslation-master/QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -2414,20 +2414,20 @@ QUEST_LV_0200_20150317_002413	You've defeated Elite Monsters as requested by Sal
 QUEST_LV_0200_20150317_002414	Talk to Graverobber Amanda
 QUEST_LV_0200_20150317_002415	Graverobber Amanda is waiting for someone's help in the Petrified City.
 QUEST_LV_0200_20150317_002416	Collect Documentaries of Ruklys' Era
-QUEST_LV_0200_20150317_002417	The thief, Amanda, asked you to collect the records of the era of Ruklys at the rendezvous point for the transaction with Wilhelmina Carriot.
+QUEST_LV_0200_20150317_002417	Graverobber Amanda, asked you to collect the records of the era of Ruklys at the rendezvous point for the transaction with Wilhelmina Carriot.
```
Comment (ttgmichael) [inline EnglishTranslation-master/QUEST_LV_0200.tsv:2417]: Lose the comma after Amanda

File: EnglishTranslation-master/QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -7380,13 +7380,13 @@ QUEST_LV_0200_20150714_007383	The information should sometimes be leaked, hid or
...
 QUEST_LV_0200_20150714_007389	I personally think the curse of the petrification is a chemical reaction.{nl}Which means you can't restore the things that are once petrified before.{nl}
 QUEST_LV_0200_20150714_007390	Even if, Bokor over there tries his best, he won't be able to restore it.{nl}
 QUEST_LV_0200_20150714_007391	But, if the petrification is in progress,{nl}Then I think we may able to save the rest even if we lost the parts that have already become petrified.{nl}
-QUEST_LV_0200_20150714_007392	Almost all curses disappear when the casters die.{nl}I think the curse of the petrification is the same as the case in the way of the pilgrim.{nl}
-QUEST_LV_0200_20150714_007393	But the curse descended for 600 years.{nl}This means the caster of the curse also has lived for 600 years.{nl}
+QUEST_LV_0200_20150714_007392	Almost all curses will wear off when the one who casted it dies.{nl}That's why I think the curse of the petrification is similar to Pilgrim's Way.{nl}
```
Comment (ttgmichael) [inline EnglishTranslation-master/QUEST_LV_0200.tsv:7388]: That's why I think the curse of the petrification is similar to (the curse) at Pilgrim's Way.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**